Corona-Warn-App/src/main/AndroidManifest.xml

```diff
@@ -25,6 +25,22 @@
 
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
 
+    <uses-permission android:name="android.permission.CAMERA" />
+
+    <uses-feature android:name="android.hardware.camera.any" />
+    <uses-feature
+        android:name="android.hardware.camera.autofocus"
+        android:required="false" />
+    <uses-feature
+        android:name="android.hardware.camera.flash"
+        android:required="false" />
+    <uses-feature
+        android:name="android.hardware.screen.landscape"
+        android:required="false" />
+    <uses-feature
+        android:name="android.hardware.wifi"
+        android:required="false" />
+
```

Comment:
Before it was added by embedded zxing see [AndroidManifest](https://github.com/journeyapps/zxing-android-embedded/blob/master/zxing-android-embedded/AndroidManifest.xml) , considering comment about Android 4 and we support from Android 6 already